src/ui/components/ExplorerHome/ExplorerTimeline.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import React, {type Node as ReactNode} from "react";
+import {extent} from "d3-array";
+import {scaleLinear} from "d3-scale";
+import {line} from "d3-shape";
+import {type Grain} from "../../../core/ledger/grain";
+
+const legendTextStyles = {
+  fontSize: "10px",
+  fill: "#fff",
+};
+const CRED_COLOR = "#6174CC";
+const GRAIN_COLOR = "#FFAA3D";
+type ExplorerTimelineProps = {|
+  +timelines: {
+    cred: Array<number>,
+    grain?: Array<Grain>,
+  },
+  +width?: number,
+  +height?: number,
+  +hasLegend?: boolean,
+  +responsive?: boolean,
+|};
+const ExplorerTimeline = (props: ExplorerTimelineProps): ReactNode => {
+  if (props.timelines.cred == null && props.timelines.grain == null) {
+    return <></>;
+  }
+  const grainExists = props.timelines.grain && true;
+  const width = props.width || 300;
+  const height = props.height || 25;
+  const viewBox = `0 0 ${width} ${height}`;
+  const intervals = props.timelines.cred.length;
+  let grainAsNumber, range, grainValues;
+  if (grainExists) {
+    const grain = props.timelines.grain || [];
+    grainAsNumber = grain.map((g) => {
+      return Number(g);
+    });
+    const credAndGrain = props.timelines.cred.concat(grainAsNumber);
+    range = extent(credAndGrain);
```

Comment:
Is it possible to have 2 different ranges for cred and grain? We can't really expect that one won't be super inflated compared to the other, which would cause the less inflated one to appear like a straight line at the bottom if they share a range.

Reply:
That was our first approach and we ran into some poblems with that approach - because cred's range did not start at 0, it put the grain line below the x-axis. I thought about just simply setting the lower limit of the range to 0 but we opted for this approach instead. We'll give it another try, though!